<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build AOSP Image</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="factoryImageFilePathHorizontalLayout">
      <item>
       <widget class="QLabel" name="factoryImageFilePathLabel">
        <property name="text">
         <string>factoryImageFilePath: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="factoryImageFilePathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="factoryImageFilePathPushButton">
        <property name="text">
         <string>choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aospBuildImgFilePathListHorizontalLayout">
      <item>
       <widget class="QLabel" name="aospBuildImgFilePathListLabel">
        <property name="text">
         <string>aospBuildImgFilePathList: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aospBuildImgFilePathListLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aospBuildImgFilePathListPushButton">
        <property name="text">
         <string>choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="logTextBrowser"/>
    </item>
    <item>
     <widget class="QPushButton" name="buildPushButton">
      <property name="text">
       <string>build</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
